src/Packetery/Core/Api/Soap/Response/PacketStatus.php

```diff
@@ -69,7 +69,7 @@ public function getStoredUntil(): ?\DateTimeImmutable {
 	 * @return void
 	 */
 	public function setStoredUntil( ?string $storedUntil ): void {
-		$formatedStoredUntil = $storedUntil ? \DateTimeImmutable::createFromFormat(
+		$formatedStoredUntil = null !== $storedUntil ? \DateTimeImmutable::createFromFormat(
```

Comment:
pro lepší přehlednost bych navrhoval něco jako:

`public function setStoredUntil(?string $storedUntil): void {
    if ($storedUntil === null) {
        $this->storedUntil = null;
        return;
    }

    $formattedStoredUntil = \DateTimeImmutable::createFromFormat(CoreHelper::MYSQL_DATE_FORMAT, $storedUntil);
    $this->storedUntil = $formattedStoredUntil instanceof \DateTimeImmutable ? $formattedStoredUntil : null;
}`